run-tests.sh

```diff
@@ -7,7 +7,19 @@
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
+# Usage:
+#   env DB=postgresql ./run-tests.sh
 
-check-manifest --ignore ".travis-*" && \
-sphinx-build -qnNW docs docs/_build/html && \
+# Quit on errors
+set -o errexit
+
+# Quit on unbound symbols
+set -o nounset
+
+python -m check_manifest --ignore ".*-requirements.txt"
+python -m sphinx.cmd.build -qnNW docs docs/_build/html
+docker-services-cli up ${DB}
```

Comment:
Minor: FYI @lnielsen, [according to the old `.travis.yml`](https://github.com/inveniosoftware/invenio-oauthclient/blob/master/.travis.yml#L18) we needed Redis so this should be `docker-services-cli up redis ${DB}`, but actually all the whole build passes without it...

Reply:
Wow, I hadn't realized at all. Good question.

Reply:
This is probably a mistake in the old Travis file......I'm trying to remember why oauthclient would need it.